AudioAnalytics/AudioAnalyzer/pom.xml

```diff
@@ -43,20 +38,23 @@
       <artifactId>azure-client-runtime</artifactId>
       <version>1.6.8</version>
     </dependency>
+    <!-- https://mvnrepository.com/artifact/com.azure/azure-storage-blob -->
```

Comment:
I would use the azure-sdk-bom instead. So it'll pull in compatible references. https://search.maven.org/artifact/com.azure/azure-sdk-bom/1.0.3/pom

Reply:
According to your comments, it has been fixed in the new version.

Reply:
I don't see it in this PR. It shows that we are still referencing the explicit packages.

Reply:
Sorry for missing this comment, now has been fixed in the new version。